hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/parsing/FhirPayloadExtractor.java

```diff
@@ -18,17 +18,21 @@
     private static final JSONParser jsonParser = new JSONParser(JSONParser.DEFAULT_PERMISSIVE_MODE);
 
     @Handler
-    public static String extractFhirPayload(Exchange exchange,String fhirMessage) throws ParseException, UnsupportedEncodingException {
-    	
-    	String methodName = "extractFhirPayload";
-        JSONObject fhirMessageJSON = (JSONObject) jsonParser.parse(fhirMessage);
-        
-        FHIRJsonMessage encodedExtractedMessage = FHIRJsonUtil.parseJson2FHIRMsg(fhirMessageJSON); // get the data property
+    public static String extractFhirPayload(Exchange exchange,String fhirMessage) throws CustomHNSException {
+    	try {
+    		String methodName = "extractFhirPayload";
```

Comment:
This should use the new Util.getMethodName().

Reply:
I had not replaced all the method names as wanted to see if we agree with method to get name. If yes, I will replace everywhere in the codebase.

Reply:
Your method looks good. The only thing I was wondering is it we wanted to have an additional utility method to handle some of the logging we always do (like starting with method name, then transaction id). There's a lot of very similar log statements.